HW456/Client.cs

```diff
@@ -0,0 +1,64 @@
+public class Client
+{
+    /// <summary>
+    /// Реализовать классы для описания клиентов двух типов: ИП и ООО.
+    /// Каждый тип клиента имеет идентификатор, основной телефон, сумма заказа.
+    /// У ИП указывается ФИО, дата рождения. У ООО указывается название, банковский счет.
+    /// Реализовать метод, который возвращает отформатированное название и сумму заказа.
+    /// </summary>
+
+    public int id;
+    public string phone;
+    public int orderSum;
+
+    public virtual string Info
+    {
+        get
+        {
+            return id + '\n' + phone + '\n' + orderSum.ToString() + "руб" + '\n'; 
+        }
+        
+    }
+
+    public static void func()
+    {
+        IE ieClient = new IE();
+        LLC llcClient = new LLC();
+      
+        ieClient.id = 65482631;
+        ieClient.phone = "+7(499)568945";
+        ieClient.orderSum = 512;
+        ieClient.date = DateOnly.FromDateTime(DateTime.Now.AddYears(-30));
+        ieClient.name = "Блабла Бла Бла";

```

Comment:
Можно же в одну инструкцию обойтись:
```suggestion
        IE ieClient = new IE
        {
            id = 65482631,
            phone = "+7(499)568945",
            orderSum = 512,
            date = DateOnly.FromDateTime(DateTime.Now.AddYears(-30)),
            name = "Блабла Бла Бла",
        };

```
И ниже так же

Reply:
поправила